### ISSUE TITLE
fix the environment issues and document solution

### ISSUE THREAD

**Issue by mkoohafkan** [mkoohafkan/arcpy#4]
There are some issues with directly using the environment as cloned from within the ArcGIS Pro application. No idea why but something about the way ArcGIS Pro clones the environment doesn't play nice with `reticulate`. It appears that there are two solutions:

1. After cloning the environment within ArcGIS Pro, clone it *again* from the anaconda prompt and then use that environment (the base arcpy environment doesn't show up in the environment listing, but maye you can clone it directly by providing a full path).
2. Following [this ESRI documentation](https://developers.arcgis.com/python/guide/understanding-conda/), create an environment from scratch and install `arcpy` and/or `argis` modules directly from the ESRI conda channel. One issue with this is that you need to install the module version from the channel that matches with your ArcGIS Pro installation, i.e., `conda install --name <conda env> -c esri arpy=<Pro Version>` 

Preliminary tests suggest that option 1 works.

**Comment by mkoohafkan** [comment]
We can do this entirely in reticulate. For example, here's the code for ArcGIS Pro v2.8:

```
library(reticulate)
conda_create("gis-envir", "arcpy==2.8",
  channel = "esri", python_version = "3.7")
```

I don't know of a good way to verify the ArcGIS Pro version or the Python version that is needed,  it will have to be up to the user to figure out.

**Comment by mkoohafkan** [comment]
implemented in 39738dc
